### ISSUE TITLE
Move shortcuts into its own module with more options. 

### ISSUE THREAD

**Issue by creesch** [creesch/reddit-moderator-toolbox#397]
Basically what I would like to add is a bunch of functionality: 

- Sorting shortcuts. 
- Ability to add icons. Basically offer a basic set which can be selected from a sort of dropdown. 
- Ability to have submenu's (much like RES has but RES doesn't allow non-reddit links). 
- Allow shortcuts to have the same name. 
- Usage of tokens. 
- Maybe but not sure: Ability to add toolbox functionality trigger in there. For example, activate queue tools. 

closing #329 and #314 in favor of this one since it falls under the scope of a new module.

**Comment by Dakta** [comment]
Important question: should shortcuts really be their own module if they only show up in the modbar? It makes more sense to me that they be organized as a feature of the modbar, since that's how they're accessed.

**Comment by agentlame** [comment]
Well, they need features and settings.  Like being able to reorder them and set them to `target="_blank"`.  

If you think it's cleaner to keep them in, then do so.

**Comment by agentlame** [comment]
Actually, this issue was opened a log time before modbar was its own module, so maybe they shouldn't be.  But setting them should be moved.

**Comment by Dakta** [comment]
Yep, I'll consider that as part of the cleanup associated with #607 and they'll get moved into Modbar's settings as a proper generic setting type.

**Comment by Dakta** [comment]
Fixed by c957f7b812d2589de5921247eff1f3be15393956
